notebooks/Vid2Desc_Gemini_1_0_Pro_Vision.ipynb

```diff
@@ -0,0 +1,378 @@
+{
```

Comment:
<p>It's a single-shot caption summarizing the video, yeah?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-diver/Vid2Persona/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
yes it is!